pages/general_rules/Procedure.tex

```diff
@@ -126,6 +126,18 @@ \subsection{Start Signal}\label{rule:start_signal}
     This rule ensures fairness and flexibility while addressing unavoidable physical challenges during league operations.
 \end{enumerate}
 
+\subsection{Restart Rule}
+\label{rule:restart}
+
+During any test, teams are allowed to perform a single restart per attempt under the following conditions:
+\begin{itemize}
+    \item A restart may only be requested until the robot completes a major scoring action.
+    \item The test time will continue running until the team has exited the \Arena{} with the robot.
+    \item A restart incurs a penalty of \SI{30}{\second}, subtracted from the total time.
```

Comment:
Subtracted from the total time is a bit confusing should be clear by the word penalty tho...

Maybe subtract from the time limit?